tests/framework/tests

```diff
@@ -209,6 +209,7 @@
   type = 'RavenFramework'
   input = 'test_external_reseed.xml'
   output = 'externalModel/testPrintHistorySet_dump_0.csv externalModel/testPrintHistorySet_dump_1.csv externalModel/testPrintHistorySet_dump_2.csv'
+  prereq = testExternalModel
```

Comment:
This modification is already in dataobject-rework ...why no conflict is shown??? O_o

Reply:
If the change is identical, I think maybe it can handle coinciding both effects.

Reply:
Let me rebase to assuage those concerns.